src/nexus-prisma/NexusPrismaBuilder.ts

```diff
@@ -281,71 +327,6 @@ export class NexusPrismaBuilder {
     }, {})
   }
 
-  protected buildSchemaForCRUD(
```

Comment:
Bringing this inline removes a level of indirection. I am struggling to follow the layers and didn't feel this one brought much value.

Reply:
Agreed! As you mentioned before, we can probably refactor the `forEach`, which will simplify things even more